README.md

```diff
@@ -30,3 +23,10 @@ See [here](docs/SETTINGS.md) for more information on the settings in the app.
 ![Screenshot](https://github.com/soda3x/ArmaReforgerServerTool/raw/main/servertoolscreen4.png)
 
 ![Screenshot](https://github.com/soda3x/ArmaReforgerServerTool/raw/main/servertoolscreen2.png)
+
+## Discord
+
+Come discuss with fellow users, seek help etc. on the Discord server [here](https://discord.gg/RmTH7gKdfK)
+
+## Server Paramters
```

Comment:
Small typo

Reply:
Done in https://github.com/soda3x/ArmaReforgerServerTool/pull/50/commits/101c683f9ee850fea381138687373d08f27d07c7.